fieldservice/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Field Service",
     "summary": "Manage Field Service Locations, Workers and Orders",
-    "version": "14.0.1.17.0",
+    "version": "14.0.1.18.0",
```

Comment:
no need to update the version as this will be done automatically when merged

Reply:
ok did the same for #917 and #918